roles/configure_jenkins_projects/tasks/main.yml

```diff
@@ -5,7 +5,7 @@
     state: directory
     owner: jenkins
     group: jenkins
-    mode: '0740'
+    mode: '0755'
```

Comment:
I can understand 0740 -> 0750, but why does this directory have to be world readable? Only the jenkins user is supposed to work with the content of this dir, right?